### ISSUE TITLE
FE/UX : [LOGS] No easy way to go back to list view after applying different parms

### ISSUE THREAD

**Issue by nityanandagohain** [SigNoz/signoz#3150]
https://github.com/SigNoz/signoz/assets/26831659/db1e734f-a094-402c-bcb5-7acdd94b8e41

**Comment by pranay01** [comment]
@nityanandagohain In current implementation, if user removes the attributes in groupby field, the list view becomes accessible. We also shows this on hove on the disabled list view. 

Doesn't this solve the issue? I don't think they need to remove filters to access list view

![listview](https://github.com/SigNoz/signoz/assets/504541/dc59531a-731e-4d46-9d6a-15da4d44b70a)

**Comment by kaushikmacharla07** [comment]
Its better not to show the "List View" tab completely in this case rather than explaining why something cannot be shown being the very first tab there.

**Comment by nityanandagohain** [comment]
Yes @pranay01  it does solve it, just then removing 5 attributes in groupby is 5 clicks. There is no way to clean it up instantly and go back to list view in a single click.

**Comment by pranay01** [comment]
> Its better not to show the "List View" tab completely in this case rather than explaining why something cannot be shown being the very first tab there.

So, time series view comes into picture only when someone selects attributes in groupby field. Otherwise list view is the default view. Not showing the tab won't be a great experience IMO

**Comment by pranay01** [comment]
> just that removing 5 attributes in groupby is 5 clicks

@nityanandagohain My sense is that mostly people will have 1 or 2 attributes in groupby field. Do you expect any common situation where people will add 5 attributes in group by field?

**Comment by ankitnayan** [comment]
seems uncommon to me too. We can wait for this improvement IMO

**Comment by nityanandagohain** [comment]
Yeah we can hold this fow now, and come up with ideas onces users start using it

**Comment by ankitnayan** [comment]
@nityanandagohain is this still relevant?

**Comment by nityanandagohain** [comment]
Yes Ankit, this is still relevant. There is no way to clear the current aggregate query for moving to list view without removing everything manually

**Comment by srikanthccv** [comment]
This is fixed now, the relevant context carried b/w the views
